kpops/component_handlers/helm_wrapper/helm_diff.py

```diff
@@ -32,13 +32,16 @@ def calculate_changes(
             # get corresponding dry-run release
             new_resource = new_release_index.pop(current_resource.filepath, None)
             yield Change(
-                current_resource.manifest,
-                new_resource.manifest if new_resource else KubernetesManifest(),
+                current_resource.manifest.model_dump(),
+                new_resource.manifest.model_dump() if new_resource else {},
```

Comment:
@disrupted Here is where I return an empty dictionary. Before it was `KubernetesManifest()`. Now Pydantic will throw a `ValidatinError`